apps/remix-ide/src/app/ui/landing-page/landing-page.js

```diff
@@ -357,6 +398,37 @@ export class LandingPage extends ViewPlugin {
       query.update({ appVersion: '0.7.7' })
       document.location.reload()
     }
+
+    const migrate = async () => {
+      tooltip('migrating workspace...')
+      try {
+        const workspaceName = await migrateToWorkspace(this.fileManager, this.filePanel)
+        tooltip('done. ' + workspaceName + ' created.')
+      } catch (e) {
+        return tooltip(e.message)
+      }
+    }
+
+    const migrateWorkspace = async () => {
+      modalDialog(
```

Comment:
Clicking outside modal does not close the modal dialog. And when i click the `X` icon to close modal, i get a success message in the tooltip saying `done. workspace_migrated_1616584411034 created.`

Reply:
In other pr